recipe/meta.yaml

```diff
@@ -52,6 +49,11 @@ requirements:
     - packaging >=21
     - lazy_loader >=0.2
   run_constrained:
+    # numpy's run_export sets the lower bound to be 1.19
+    # https://github.com/conda-forge/numpy-feedstock/issues/324
+    # We must set the lower bound in accordance to scikit-image's
+    # runtime requirements
+    - numpy >=1.23
```

Comment:
Should this just go in `run`?

Reply:
No. Otherwise it will overwrite the upper bound too and set the upper bound to be infinite.

Reply:
Actually these just become additional lines in `depends` (including `run_exports`). So collectively they specify `numpy`'s constraint correctly

Tried to explain this in more detail with an example from `scipy` where this is already used: https://github.com/conda-forge/numpy-feedstock/issues/324#issuecomment-2294512688

Please let me know in that thread if you have more questions on that

Reply:
i guess maybe something changed in how conda/mamba handle repeated dependencies over the years.

It used to be very poorly supported and only one of the lines would get picked up.

Reply:
i can remember to try to adjust this later, but i'm almost too scared to try.